panedr/panedr/__init__.py

```diff
@@ -1,5 +1,9 @@
 # -*- coding: utf-8 -*-
-from .panedr import edr_to_df, get_unit_dictionary
-import pbr.version
+from .panedr import ENX_VERSION, edr_to_df, get_unit_dictionary  # noqa: E402
+import pbr.version  # noqa: E402
```

Comment:
```suggestion
import pbr.version
from .panedr import ENX_VERSION, edr_to_df, get_unit_dictionary
```
You might have to provide some more information as to why these are necessary. I can't reproduce E402 errors using flake8 or pylint.

I do get a pylint C0411 (wrong-import-order), but that's easily fixable.

Reply:
That was some left over from times when there were some lines in front to set the path. Removed unnecessary `noqa` statements and reordered imports.